### PR TITLE
Milestone 8 : Details modal (pair programming)

### PR DESCRIPTION
**Pull request by Chrissiku**
In this milestone of the project, we used the popup window in order to hold post details of the website and that dynamically in JavaScript.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,14 +12,64 @@
...
+                See Live&nbsp;&nbsp;<i class="fa-solid fa-power-off"></i>
+              </button>
+              <button type="button" class="btn">
+                See Source&nbsp;&nbsp;<i class="fa-brands fa-github"></i>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (Mre55) [inline index.html]: Nice job with the popup window 👍. But as the projects require the `pop up` window should render the project detail dynamically from the `JavaScript object` where the object contains the information for each project 👇.

image

Kindly, remove this `hardcoded` data from the `index.html` file and get the data of each project from the JavaScript object and render it to the page.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @Chrissiku,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._

**[REQUIRED]** As this project is `pair-programming` you should have work with your team. Here I see your commit only to this project. Kindly, do the changes with your team and they should have to commit.

image





## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Chrissiku) [reply index.html]: ## What we changed  ♻️  
- 🆗 : Remove the hardcoded data from the `index.html` file and get the data of each project from the JavaScript object and render it to the page
- 🆗 : As this project is pair-programming we work in a team.

**Review by gemmen29 — APPROVED** [review]
Hi @Chrissiku, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**